client/app/components/preprint-basics/component.js

```diff
@@ -183,8 +177,17 @@ export default Ember.Component.extend(BASICS_VALIDATIONS, {
     },
     init(){
         this._super(...arguments);
-        this.get('store').findRecord('node', ENV.NODE_GUID).then((result)=>{
+        this.get('store').findRecord('node', ENV.nodeGuid).then((result)=>{
```

Comment:
I brought this up to date with the latest changes on the `feature/submission-form` branch, where it is `nodeGuid`: https://github.com/cos-labs/collections/blob/feature/submission-form/client/config/environment.js#L137